client-cli/src/test/java/org/eclipse/dataspaceconnector/registration/cli/ParticipantsCommandTest.java

```diff
@@ -152,4 +169,12 @@ private int executeParticipantsList(String inputCmd, String inputValue) {
                 inputCmd, inputValue,
                 "participants", "list");
     }
+
+    private int executeParticipantStatus() {
```

Comment:
This PR is already merged upstream: https://github.com/eclipse-dataspaceconnector/RegistrationService/pull/15
so instead of using deprecated serverUrl as an input `-s` we can start using did `-d`

Reply:
Updated , Thanks.